source/hardware/controllers/hercules_djcontrol_inpulse_500.rst

```diff
@@ -0,0 +1,316 @@
+:hwlabel:`SHIFT` + :hwlabel:`SYNC` button                        Match key to currently playing track.
+:hwlabel:`CUE` button                                            Specifies, plays or recalls temporary cue point.
+:hwlabel:`SHIFT` + :hwlabel:`CUE` button                         Return to the beginning of the track.
+:hwlabel:`Play` button                                           Play/Pause the current track.
+:hwlabel:`SHIFT` + :hwlabel:`Play` button                        Cue Stutter.
+:hwlabel:`VINYL` button                                          Toggle scratch mode (default: on)
+:hwlabel:`SHIFT` + :hwlabel:`VINYL` button                       Toggle vinyl pause effect (default: off)
+Loop :hwlabel:`IN` button                                        Mark start of the loop.
+:hwlabel:`SHIFT` + Loop :hwlabel:`IN` button                     Jump to loop start.
+Loop :hwlabel:`OUT` button                                       Mark end of the loop.
+:hwlabel:`SHIFT` + Loop :hwlabel:`OUT` button                    Jump to loop end.
+Beat Align LED                                                   Track alignment markers (make sure :hwlabel:`BEATMATCH GUIDE` is on for this to work).
+Loop encoder push                                                Reloop toggle
+Loop encoder turn                                                Double or halve current loop
+:hwlabel:`SHIFT` + Loop encoder                                  4-bar loop enable
+:hwlabel:`SLIP`                                                  Toggles splip mode.
+:hwlabel:`QUANT`                                                 Toggles quantize.
+:hwlabel:`SHIFT` + :hwlabel:`QUANT`                              Toggles keylock.
+===============================================================  ==========================================
+
+Browser
+~~~~~~~
+
+===============================================================  ==========================================
+Control                                                          Function
+===============================================================  ==========================================
+:hwlabel:`BROWSER` encoder (turn)                                Move up/down list.
+:hwlabel:`SHIFT` + :hwlabel:`BROWSER` encoder (turn)             Move left/right (fold/unfold) list.
+:hwlabel:`BROWSER` encoder (press)                               Switch focus between list and file view.
+:hwlabel:`SHIFT` + Encoder (press)                               Maximize/Minimize library view.
+:hwlabel:`ASSISTANT` button                                      Toggle AutoDJ (be sure a playlist was created for AutoDJ before activating this function).
+===============================================================  ==========================================
+
+FX
+~~
+
+.. note::
+   This mapping turns the FX button into channel selection,
+   relegating FX to pad mode 7. The LED indicates which
+   channel is active on the corresponding deck
```

Comment:
I don't understand what this does.
Assign decks to fx units?

Reply:
It effectively uses (abuses?) the FX buttons as deck selectors, so you can use the controller as a 4 deck controllers.

It is a bit strange, but it works very intuitively.

Reply:
> FX1,2,3,4, but instead refer to Deck 1,2,3,4

refert to Deck 3, 1, 2, 4?